docs/01-overview/02-main-areas/service-management/smgt-04-sc-provisioning.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Provisioning and binding flow
+---
+
+The diagram shows an overview of interactions between all resources related to the processes of provisioning and binding:
+
+![Provisioning and binding](./assets/provisioning-and-binding.svg)
+
+## Provision a service
+
+Provisioning a service means creating an instance of a service. To provision a service, create a ServiceInstance custom resource. During this process, the Service Broker creates a new instance of a service. The form and scope of this instance depends on the Service Broker.
+
+![Provisioning](./assets/provisioning.svg)
+
+## Bind a service
```

Comment:
same here, it's somehow instructional rather than explanatory. However, not sure how many steps that'll have?

Reply:
It will be a very short tutorial, but let's go with this approach 🙂